SECURITY.md

```diff
@@ -0,0 +1,55 @@
+# Security Policy
+
+## Supported Versions
+
+We commit to publishing security updates for the version of Flutter currently
+on the `stable` branch.
+
+## Reporting a Vulnerability
+
+To report a vulnerability, please e-mail `security@flutter.dev` with a description of the issue,
+the steps you took to create the issue, affected versions, and if known, mitigations for the issue.
+
+We should reply within three working days, probably much sooner.
+
+We use GitHub's security advisory feature to track open security issues. You should expect
+a close collaboration as we work to resolve the issue you have resolved. Please reach out to
+`security@flutter.dev` again if you do not receive prompt attention and regular updates. You
+may also reach out to the team via our public [Chat] channels; however, please make sure to
```

Comment:
Are we sure we want this?  It might be good to keep security issues under wraps (modulo the behavior of the person who discovers the issue when they discover it) until we decide to publish them.  e.g. I think Chrome does this.

Reply:
Which part?

Reply:
Assuming you mean the mention of chat channels, I've adjusted this to: `You may also reach out to the team via our public [Discord] chat channels; however, please make sure to e-mail `security@flutter.dev` when reporting an issue, and avoid revealing information about vulnerabilities in public if that could put users at risk.`